tests/fafnir_tests/mod.rs

```diff
@@ -338,7 +540,7 @@ pub fn main_test(mut es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapp
     // Test that the coord property of a POI has been well loaded
     // We test latitude and longitude
     let ocean_poi = &ocean_place.poi().unwrap();
-    assert_eq!(&ocean_poi.id, "osm:node:5589618289");
+    assert_eq!(&ocean_poi.id, "osm:node:55896182890");
```

Comment:
I wonder why we have '0' being added at the end...

Reply:
that's because `global_id_from_imposm` is called twice, first in `layer_poi` (unused), and then a second time in the main query with `osm_id` as input (which is actually a osm_hash, hence the bug).

You should use `global_id` (returned by `layer_poi`) directly in src/lib.rs line 376  
https://github.com/QwantResearch/fafnir/pull/32/files#diff-b4aea3e418ccdb71239b96952d9cddb6R376